### DISCUSSION TITLE
[Script request] GLPI

### DISCUSSION THREAD

**Discussion by opastorello** [community-scripts/ProxmoxVE#1024] [Request script]
### Application Name

GLPI

### Website

https://glpi-project.org

### Description

GLPI (Gestionnaire Libre de Parc Informatique) is an open-source IT asset management and service management software designed to help organizations efficiently manage their IT infrastructure and related services. It provides a comprehensive suite of tools to track hardware, software, and manage support tickets, making it an essential tool for IT administrators and service desks.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by tremor021** [comment]
#1201
